src/git.js

```diff
@@ -0,0 +1,36 @@
+const childProcess = require('child_process')
+
+const SEPARATOR = '/////'
+const GIT_FORMAT_SPEC = `--pretty=format:"%an${SEPARATOR}%h${SEPARATOR}%s"`
+
+function getGitLogOutput (tagPrefix) {
+  const commitListSpec = `$(git tag | grep '^${tagPrefix}' | tail -n2 | head -n1)..HEAD`
```

Comment:
Instead of exec shell command, it can be more easy to read and maintain with native git binding like: https://github.com/nodegit/nodegit

Reply:
I knew about this module, but I didn't include it, because what we are doing with git here is very simple, and doesn't justify to include such a heavy project (many dependencies, long history, native dependencies which add significant time to npm install, etc...)